Podfile

```diff
@@ -1,5 +1,5 @@
 source 'https://cdn.cocoapods.org/'
-use_frameworks!
+use_frameworks! :linkage => :static
```

Comment:
Adds configuration to include Yoga inside the xcframework, consistent with the existing wrapping structure.
This change does not affect environments using CocoaPods or Swift Package Manager.